stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api/GsonChatApi.kt

```diff
@@ -374,29 +374,14 @@ internal class GsonChatApi(
         channelId: String,
         query: QueryChannelRequest,
     ): Call<Channel> {
-
-        if (connectionId.isEmpty()) {
-            return noConnectionIdError()
-        }
-
-        return if (channelId.isEmpty()) {
-            retrofitApi.queryChannel(
-                channelType,
-                apiKey,
-                userId,
-                connectionId,
-                query
-            )
-        } else {
-            retrofitApi.queryChannel(
-                channelType,
-                channelId,
-                apiKey,
-                userId,
-                connectionId,
-                query
-            )
-        }.map(::flattenChannel)
+        return retrofitApi.queryChannel(
```

Comment:
There was `channelId` checking and you deleted it. Why it's not required anymore? I don't see that you move this checking elsewhere.

Reply:
My bad, fixed.